charts/function-mesh-operator/README.md

```diff
@@ -1,16 +1,16 @@
 # function-mesh-operator
 
-![Version: 0.2.7](https://img.shields.io/badge/Version-0.2.7-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.7.0](https://img.shields.io/badge/AppVersion-0.7.0-informational?style=flat-square)
+![Version: 0.2.8](https://img.shields.io/badge/Version-0.2.8-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.8.0](https://img.shields.io/badge/AppVersion-0.8.0-informational?style=flat-square)
 
-The Function Mesh operator Helm chart for Kubernetes
+function mesh operator Helm chart for Kubernetes
 
 **Homepage:** <https://github.com/streamnative/function-mesh>
 
 ## Maintainers
 
-| Name | Email                                | URL |
-| ---- |--------------------------------------| --- |
-| Function Mesh | mailto:function-mesh@streamnative.io | https://github.com/streamnative/function-mesh |
+| Name | Email | Url |
```

Comment:
I think we can follow @Huanli-Meng 's suggestions, pls refer to this: https://github.com/streamnative/function-mesh/pull/497#pullrequestreview-1161363574